hmpps-prisoner-search-indexer/src/main/resources/db/migration/V1_2__prisoner_differences_add_column.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE prisoner_differences ADD COLUMN label varchar(10) not null default 'GREENBLUE';
+
+CREATE INDEX prisoner_differences_label_idx on prisoner_differences(label);
```

Comment:
can't see this index being used or making any difference?  I'd have thought it would be better to modify the existing date time index to add in the label instead, since that matches the change to the repository method.

Reply:
Done